rlp/raw_test.go

```diff
@@ -65,10 +65,10 @@ func TestSplitTypes(t *testing.T) {
 		t.Errorf("SplitString returned %q, want %q", err, ErrExpectedString)
 	}
 	if _, _, err := SplitList(unhex("01")); err != ErrExpectedList {
-		t.Errorf("SplitString returned %q, want %q", err, ErrExpectedList)
+		t.Errorf("SplitList returned %q, want %q", err, ErrExpectedList)
```

Comment:
IMO if you want to fix up the test, it should be rewritten even more. And we don't need to mention exactly what method was called, the test will tell us what line it was that failed. 

```golang
if _, _, err := SplitList(unhex("01")); !errors.Is(err, ErrExpectedList) {
              t.Errorf("have %v want %v", err, ErrExpectedList)
}
...
```

Reply:
OK, I have reimplemented the testing method of SplitString and SplitList seperately.